src/v2/BaseLoginRouter.js

```diff
@@ -245,6 +249,32 @@ export default Router.extend({
     this.controller.render();
   },
 
+  /**
+    * When "Remember My Username" is enabled, we save the identifier in a cookie
+    * so that the next time the user visits the SIW, the identifier field can be 
+    * pre-filled with this value.
+   */
+  updateIdentifierCookie: function(idxResponse) {
```

Comment:
every idx response will call `updateIdentifierCookie` even when it is not required.

Reply:
The very first step in that method checks for successful authentication so it will return early, but I can move that check outside the method instead for clarity.